shelf_router/lib/src/router.dart

```diff
@@ -115,18 +115,17 @@ class Router {
     _routes.add(RouterEntry('ALL', route, handler));
   }
 
-  /// Mount a router below a prefix.
+  /// Mount a handler below a prefix.
   ///
   /// In this case prefix may not contain any parameters, nor
-  void mount(String prefix, Router router) {
+  void mount(String prefix, Handler handler) {
```

Comment:
You are arguing that this isn't a breaking change because you added `Router.call` ?

Reply:
Yes with a `call` method, any `Router` becomes a `Handler` (since handler is just a function definition).

```
Handler handler = Router();
```